CHANGELOG.md

```diff
@@ -1,13 +1,10 @@
 # Changelog
-## [Unreleased]
 
-### Added
+## [0.5.0]
 
 ### Changed
-
-### Fixed
-
-### Deprecated
```

Comment:
Don't change the "Unreleased" section as it provides some sort of template.

Reply:
thanks, fixed